README.md

```diff
@@ -1,61 +1,191 @@
 
-### How to install the emitter - SOURCE
+## Install
 
-On the source cluster, copy the file respective emitter JAR to _/usr/lib/hive/lib/_ and configure _/etc/hive/conf/hive-site.xml_ with the properties required by the listener to talk to the messaging infrastructure:
+1. [Download the TGZ](https://repository.sonatype.org/service/local/artifact/maven/redirect?r=central-proxy&g=com.hotels&a=shunting-yard&p=tgz&v=RELEASE&c=bin) from Maven central and then uncompress the file by executing:
 
-#### SQS
-    hive.metastore.event.listeners = com.hotels.shunting.yard.event.emitter.sqs.listener.SqsMetaStoreEventListener
-    com.hotels.shunting.yard.event.emitter.sqs.queue = https://sqs.<region>.amazonaws.com/<account-id>/<topic-name>-queue.fifo
-    com.hotels.shunting.yard.event.emitter.sqs.group.id = <group-id>
+```
+tar -xzf shunting-yard-binary-<version>-bin.tgz
+```
+    
+Although it's not necessary, we recommend exporting the environment variable `SHUNTING_YARD_HOME` by setting its value to wherever you extracted it to:
 
-Once this is done restart Hive:
+```
+export SHUNTING_YARD_HOME=/<foo>/<bar>/shunting-yard-<version>
+```
 
-    sudo restart hive-hcatalog-server
+2. Download the latest version of [Circus Train](https://repository.sonatype.org/service/local/artifact/maven/redirect?r=central-proxy&g=com.hotels&a=circus-train&p=tgz&v=RELEASE&c=bin) and uncompress it:
 
-### How to install the receiver - TARGET
+```
+tar -xzf circus-train-<version>-bin.tgz
+```
 
-On the target cluster, download and install the latests version of Circus Train and set the `CIRCUS_TRAIN_HOME` environment variable:
+Set the `CIRCUS_TRAIN_HOME` environment variable:        
 
-    export CIRCUS_TRAIN_HOME=/home/hadoop/circus-train-<circus-train-version>
+```
+export CIRCUS_TRAIN_HOME=/<foo>/<bar>/circus-train-<circus-train-version>
+```
 
-Export the location of the Hive libraries:
+## Usage
+To run Shunting Yard you just need to execute the `bin/replicator.sh` script in the installation directory and pass the configuration file: 
+
+```
+$SHUNTING_YARD_HOME/bin/replicator.sh --config=/path/to/config/file.yml
+```
+
+### EMR
+If you are planning to run Shunting Yard on EMR you will need to set up the EMR classpath by exporting the following environment variables before calling the `bin/replicator.sh` script:
+
+```
+export HCAT_LIB=/usr/lib/hive-hcatalog/share/hcatalog/
+export HIVE_LIB=/usr/lib/hive/lib/
+```
 
-    export HIVE_LIB=/usr/lib/hive/lib/
-    export HCAT_LIB=/usr/lib/hive-hcatalog/share/hcatalog/
+Note that the paths above are correct as of when this document was last updated but may differ across EMR versions. Refer to the [EMR release guide](http://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-release-components.html) for more up to date information if necessary.
```

Comment:
up to date information &rightarrow; up-to-date information

Up to date is without hyphens when it's used after the noun it describes: "The information is up to date."
When it's before the noun, it's with hyphens: "This is an up-to-date information."
source: https://dictionary.cambridge.org/dictionary/english/up-to-date

Reply:
.... closes laptop and goes home ;)